src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -5,29 +5,39 @@
 
 'use strict'
 
+import * as os from 'os'
 import * as path from 'path'
 import * as vscode from 'vscode'
 
-import { DebugConfiguration } from '../../lambda/local/debugConfiguration'
+import { PythonDebugConfiguration } from '../../lambda/local/debugConfiguration'
+import { writeFile } from '../filesystem'
+import { fileExists, readFileAsString } from '../filesystemUtilities'
 import { LambdaHandlerCandidate } from '../lambdaHandlerSearch'
 import { getLogger } from '../logger'
 import { DefaultSamCliProcessInvoker, DefaultSamCliTaskInvoker, } from '../sam/cli/samCliInvoker'
 import { Datum } from '../telemetry/telemetryEvent'
 import { registerCommand } from '../telemetry/telemetryUtils'
-import { getDebugPort } from '../utilities/vsCodeUtils'
+import { getChannelLogger, getDebugPort } from '../utilities/vsCodeUtils'
 
 import {
     CodeLensProviderParams,
     getInvokeCmdKey,
     getMetricDatum,
     makeCodeLenses,
 } from './codeLensUtils'
-import { LambdaLocalInvokeArguments, LocalLambdaRunner } from './localLambdaRunner'
+import {
+    executeSamBuild,
+    invokeLambdaFunction,
+    LambdaLocalInvokeParams,
+    makeBuildDir,
+    makeInputTemplate,
+} from './localLambdaRunner'
 
 export const PYTHON_LANGUAGE = 'python'
 export const PYTHON_ALLFILES: vscode.DocumentFilter[] = [
     { language: PYTHON_LANGUAGE }
 ]
+// TODO: Change logger.info to logger.debug or potentially reformat/remove these logger statements before release.
```

Comment:
Be sure to complete the 'before release' TODOs before merging.

Reply:
✔